### PR TITLE
Update TS declarations

### PR DESCRIPTION
**Pull request by lukyanovas**

### CODE REVIEW

File: ts/dx.all.d.ts [hunk 1]
```diff
@@ -2169,7 +2170,9 @@ declare module DevExpress.ui {
         selectRowsByIndexes(indexes: Array<number>): Promise<any> & JQueryPromise<any>;
         /** Shows the column chooser. */
         showColumnChooser(): void;
+        /** Gets the current {WidgetName} state. */
```
Comment (San4es) [inline ts/dx.all.d.ts]: `{WidgetName}`
Looks strange to me in the member description. Replace it with "widget"?

/cc @RomanTsukanov  @viterobk

Comment (RomanTsukanov) [reply ts/dx.all.d.ts]: > Replace it with "widget"?

Done

Comment (IlyaKhD) [reply ts/dx.all.d.ts]: We're going to leave it as-is for now and fix it with another PR as this descriptions problem applies to several generators.

**Review by IlyaKhD — DISMISSED** [review]

File: ts/dx.all.d.ts (hunk 1 above)
Comment (San4es) [reply ts/dx.all.d.ts]: Ок

**Review by San4es — DISMISSED** [review]

**Review by komarovalexander — DISMISSED** [review]

**Review by IlyaKhD — APPROVED** [review]

**Pull request merged**